app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -42,31 +54,164 @@ class CollectionTest {
+            Espresso.onView(ViewMatchers.withText("Test_Page_1")).click()
+        }
+        // Page content: 1
+        browserScreen {
+            verifyPageContent("Page content: 1")
+            mDevice.pressBack() // go to main page
+        }
+
+        // tab_in_collection_item
+        homeScreen {
+            verifyHomeScreen()
+            Espresso.onView(ViewMatchers.withId(R.id.close_tab_button)).click()
+            // On homeview, expand the collection and open the first saved page
+            org.mozilla.fenix.ui.robots.mDevice.wait(
+                Until.findObject(By.text("Test_Page_2")),
+                TestAssetHelper.waitingTime)
+            Espresso.onView(ViewMatchers.withText("Test_Page_2")).click()
+        }
+
+        // Page content: 2
+        browserScreen {
+            verifyPageContent("Page content: 2")
+        }
     }
 
-    @Ignore
     @Test
-    fun OpenTabFromCollectionTest() {
-        // Open one tab from Collection in the Homescreen view
+    @Ignore("Temp disable test - see: https://github.com/mozilla-mobile/fenix/issues/5793")
+    // Rename Collection from the Homescreen
+    fun renameCollectionTest() {
+
+        val firstWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        createCollection(firstWebPage.url, "testcollection_1")
+
+        homeScreen {
+            // On homeview, tap the 3-dot button to expand, select rename, rename collection
+            clickCollectionThreeDotButton()
+            selectRenameCollection()
+            typeCollectionName("renamed_collection")
+            mDevice.wait(Until.findObject(By.text("renamed_collection")), TestAssetHelper.waitingTime)
+            // Verify the new name is displayed on homeview
+            Espresso.onView(ViewMatchers.withText("renamed_collection"))
+                .check(ViewAssertions
+                    .matches(ViewMatchers.withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+        }
     }
 
-    @Ignore
     @Test
-    fun RenameCollectionTest() {
-        // Rename Collection from the Homescreen
+    @Ignore("Temp disable test - see: https://github.com/mozilla-mobile/fenix/issues/5793")
+    // Delete Collection from the Homescreen
+    fun deleteCollectionTest() {
+
+        val firstWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        createCollection(firstWebPage.url, "testcollection_1")
+
+        homeScreen {
+            // Choose delete collection from homeview, and confirm
+            clickCollectionThreeDotButton()
+            selectDeleteCollection()
+            confirmDeleteCollection()
+
+            // Check for No collections caption
+            Espresso.onView(ViewMatchers.withText("No collections"))
+                .check(ViewAssertions
+                    .matches(ViewMatchers.withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
+        }
     }
 
-    @Ignore
+    // Open 2 webpages, and save each of them to a single collection
```

Comment:
I like these explanatory / summary notes in places where it's not 100% clear what the test objective is.  We should do this more often so that it's easier for someone else to maintain the test in the future.